docs/content/standards/kiosk-apps/nft-rental.mdx

```diff
@@ -0,0 +1,428 @@
+
+### Temporary assets and subscriptions {#temp-assets-subs}
+
+Temporary assets and subscriptions are notable applications of rental NFTs, offering accessibility to virtual experiences like high-end virtual casinos or curated digital fashion. These NFTs cater to diverse budgets, broadening audience reach. Subscription rentals extend to pools of digital assets, allowing users to pay monthly for a set number of items, fostering accessibility, user retention, and acquisition. Holders can rent out unused subscriptions, ensuring no loss for them, potential customer gains for the protocol, and a commitment-free trial for temporary holders. This showcases the adaptability and user-centric appeal of rental NFTs in diverse scenarios.
+
+## Smart contract design
+
+:::warning
+
+Transferring kiosks might result in unexpected behaviors while an asset is being rented. If you want to disallow kiosk transferring all together, consider using personal kiosks.
+
+:::
+
+The smart contract design leverages the Kiosk Apps standard to create a framework enabling transactions between lenders and borrowers. Both parties are required to install this extension to facilitate lending or borrowing activities. Moreover, the creator of the rented asset must establish a rental policy and a `ProtectedTP` object, granting authorization to the renting extension for managing assets subject to royalty enforcement.
+
+:::info
+
+This implementation is charging a rental fee based on days. You can re-purpose and update the logic to support charging per hour, or even seconds.
+
+:::
+
+## Move modules
+
+The NFT Rental example uses a single module, `nft_rental.move`. You can find the source for this file hosted in the [`sui-nft-rental` repository](https://github.com/MystenLabs/sui-nft-rental/blob/main/move/nft_rental/sources/nft_rental.move) on GitHub. The source code includes extensive comments to help you follow the example's logic and structure. 
+
+### nft_rental
+
+The `nft_rental` module provides an API that facilitates lending or borrowing through the following operations:
+
+- List for renting
+- Delist from renting
+- Rent
+- Borrow by reference and borrow by value
+- Reclaim for the lender
+
+### Structs
+
+The object model of the `nft_rental` module provides the structure of the app, beginning with the `Rentables` object. The struct only has the `drop` ability and acts as the extension key for the kiosk `Rentables` extension.
+
+```move
+struct Rentables has drop {}
+```
+
+The `Rented` struct represents a rented item. The only field the struct includes is the ID of the object. It is used as the DF key in the borrower's `Bag <key, value>` entry when someone is actively borrowing an item.
```

Comment:
nit: I would tend to prefer avoiding acronyms like "DF" in documentation like this, but would defer to @ronny-mysten .

Reply:
Also `Bag` does not have type parameters, so we shouldn't include them when describing a `Bag` type instance here.

Reply:
Yeah wanted to express in a pseudo code way the relationship of the key & the Bag but you are right this could be confusing. Would probably be better to omit the `Bag <key, value>` reference altogether and replace it just with `Bag`.